include/highfive/H5Reference.hpp

```diff
@@ -64,8 +55,8 @@ class Reference {
 
   protected:
     /// \brief Create a Reference from a low-level HDF5 object reference
-    explicit Reference(const hobj_ref_t h5_ref)
-        : href(h5_ref){};
+    inline explicit Reference(const hobj_ref_t h5_ref)
```

Comment:
Since this constructor is defined inside the class declaration, is it not automatically `inline` ?

Reply:
I thought that user-defined constructors follow same rules as functions, i.e. not inline. I did a quick search in cppreference and did not find anything on it, but Id be happy to know the fine print.

Reply:
Just for completeness: 
https://en.cppreference.com/w/cpp/language/inline (second paragraph)
`inline` is not necessary here because it's implicit (but it's not a mistake).